### DISCUSSION TITLE
downgrade java version ?

### DISCUSSION THREAD

**Discussion by maifeeulasad** [lanthale/PhotoSlide#14] [General]
don't you think, if java version is downgraded more people will be able to develop this ?

**Comment by lanthale** [comment]
Which jdk version do you think is good ? 11 or back to 8 ?

**Reply by maifeeulasad** [reply]
11 ?
This is widely used. That's why I said.

**Comment by lessthanoptimal** [comment]
You might want to look at https://github.com/bsideup/jabel that's what I use to output JRE 8 bytecode but use Java 14 syntax. I'm stuck at 8 because of Android support.

**Reply by maifeeulasad** [reply]
That's the point, when developing android application on regular basis, Java-11 is everyone's hot favorite.

**Reply by lessthanoptimal** [reply]
I had missed the memo about Android finally supporting a more modern version of Java! I thought Google was hell bent on punishing Oracle for that lawsuit and wanted to force Java people to switch to Kotlin if they wanted modern language features.

**Comment by lanthale** [comment]
I will try in this evening if all is working with jdk11 and let you know if it works.

**Reply by lanthale** [reply]
Just checked-in the necessary code changes. @maifeeulasad Can you check if the build with JDK11 is fine now ?

**Reply by maifeeulasad** [reply]
@lanthale , i encountered two errors while syncing project :
```
Cannot resolve org.boofcv:boofcv-fatjar:0.37
Cannot resolve org.openimaj:Openimajfaces:1.3.10
```
![Screenshot from 2021-02-18 01-09-56](https://user-images.githubusercontent.com/29339330/108255128-0ba74500-7186-11eb-8cad-bedc2b84e128.png)

**Reply by lanthale** [reply]
I have now deactivated the not yet finished face detected. Please try again.

**Comment by ohbus** [comment]
The industry is moving with JDK 11 so to say.

If there are are no code written strictly to JDK 11 we can keep the project to JDK 8

**Reply by lanthale** [reply]
The whole project is modularized already and therefore JDK8 is not an option. Also all newer JavaFX releases require JDK11 as minimum and I am using JDK11 language features.
